src/main/java/fun/epak/pak/service/UserService.java

```diff
@@ -88,10 +89,16 @@ public void saveChangedUser(ChangeUserDataRequest userData, MultipartFile multip
         if (userData.getPassword() != null && !"".equals(userData.getPassword())) {
             user.setPassword(passwordEncoder.encode(userData.getPassword()));
         }
-        if(!multipartFile.isEmpty()){
-            user.setImageName(user.getUsername().toLowerCase() + "." +  getFileExtension(multipartFile));
+        if (!multipartFile.isEmpty()) {
+            user.setImageName(user.getUsername().toLowerCase() + "." + getFileExtension(multipartFile));
             saveFile(multipartFile, user);
         }
         userRepository.save(user);
     }
+
+    public OtherUserProfileData loadOtherUserProfileData(long id) {
+        User user = userRepository.findById(id).orElseThrow();
+        String userImagePath = "/data/images/profiles/" + user.getId() + "/" + user.getImageName();
```

Comment:
Wyrzućmy `/data/images/profiles/` do application properties i wczytujmy to z nich